bedrock/mozorg/templates/mozorg/home/includes/brand-takeover.html

```diff
@@ -0,0 +1,26 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+<article id="brand-takeover" class="brand-takeover hidden">
+
+  <div class="header-strip">
+    <p class="content">
+      <img src="{{ static('img/pebbles/moz-wordmark-dark-reverse.svg') }}" alt="Mozilla" width="115" height="36">
+      <button type="button" id="close-takeover">{{ _('Continue to mozilla.org') }}</button>
+    </p>
+  </div>
+
+  <div class="takeover-content">
+    <h2>{{ _('Introducing the new Mozilla identity') }}</h2>
+    <p class="takeover-cta">{{ _('Learn more at <br><a rel="external" href="https://blog.mozilla.org/opendesign/">Mozilla Open Design</a>') }}</p>
```

Comment:
(I think I did this) but normally we'd make the `href` external to the string here. Seeing as this is `en` only and won't be translated, I think we're ok to leave this.